code/game/machinery/requests_console.dm

```diff
@@ -266,15 +266,15 @@ var/list/obj/machinery/requests_console/requests_consoles = list()
 				else
 					priority = -1
 		else
-			to_chat(usr, "<span class='warning'>Invalid characters found in the text.</span>")
+			to_chat(usr, "<span class='warning'>Invalid characters or no text detected.</span>")
 			dpt = "";
 			msgVerified = ""
 			msgStamped = ""
 			screen = 0
 			priority = -1
 
 	if(href_list["writeAnnouncement"])
-		var/new_message = copytext(reject_bad_text(input(usr, "Write your message:", "Awaiting Input", "")),1,MAX_MESSAGE_LEN)
+		var/new_message = copytext(stripped_message(usr, "Write your message:", "Departmental Announcement", ""),1,MAX_MESSAGE_LEN)
```

Comment:
```suggestion
		var/new_message = stripped_message(usr, "Write your message:", "Departmental Announcement", "")
```

Reply:
Isn't it done that way to enforce a character limit?

Reply:
`stripped_message` already does that

Reply:
based